### PR TITLE
Add import to wallet example and to iframe stamper

### PR DESCRIPTION
**Pull request by oliviathet**
## Summary & Motivation
* `http-client` package: Add new API with `InitImportWallet` and `ImportWallet`
* `iframe-stamper` package: Add `injectImportBundle` (injects the `importBundle` returned from `InitImportWallet` and eventually `InitImportPrivateKey`) and `extractWalletEncryptedBundle` (extracts the bundle that contains the encrypted wallet seed and the encapped public key, this resulting `encryptedBundle` is sent to `ImportWallet`)
* `examples/wallet-export`: Add import wallet to this example. Use modals for import and export. Improve React component abstraction for import/export so it's easier to read.

Linear ticket: ENG-1080

TODO:
* test e2e (import + export) after preprod v10.1->v11 migration :)

https://github.com/tkhq/sdk/assets/7981334/78e8fe1c-6c1d-49a3-b0e9-4d17d6d5b4eb

## How I Tested These Changes
Added unit tests. Tested locally against an SDK example

## Did you add a changeset?
Yes

If updating one of our packages, you'll likely need to add a changeset to your PR. To do so, run [`pnpm changeset`](https://pnpm.io/using-changesets#adding-new-changesets). `pnpm changeset` will generate a file where you should write a human friendly message about the changes. Note how this ([example](https://github.com/tkhq/sdk/blob/b409cd06790f011bf939adcf0755499b8e7497ae/.changeset/extra-http-exports.md?plain=1#L1)) includes the package name (should be auto added by the command) along with the type of [semver change (major.minor.patch)](https://semver.org/) (which you should set).

These changes will be used at release time to determine what packages to publish and how to bump their version. For more context see [this comment](https://github.com/tkhq/sdk/pull/67#issuecomment-1568838440).

### CODE REVIEW

**Review by andrewkmin — APPROVED** [review]
lgtm! 🔥🔥🔥

minor thoughts: updated `READMEs` for the package + example could be helpful

File: examples/wallet-export/src/components/ImportWallet.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+  return (
+    <div className={styles.modalInner}>
+      <div className={styles.modalDetails}>
+        <h2>Secret Recovery Phrase</h2>
+        <div className={styles.modalSpace}>
+          <p>
+            Import an existing wallet with your secret recovery phrase. Only you
+            should know your secret recovery phrase. A secret recovery phrase
+            can 12, 15, 18, 21, or 24 words.
```
Comment (andrewkmin) [inline examples/wallet-export/src/components/ImportWallet.tsx:73]: nit: `can be`

File: examples/wallet-export/.env.local.example
Comment (andrewkmin) [inline examples/wallet-export/.env.local.example:1]: should we rename this example `wallet-import-export`? 😂

File: examples/wallet-export/src/components/ImportWallet.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      setIframeDisplay("none");
+    } else {
+      throw new Error("failed to import wallet");
+    }
+  };
+
+  return (
+    <div className={styles.modalInner}>
+      <div className={styles.modalDetails}>
+        <h2>Secret Recovery Phrase</h2>
```
Comment (andrewkmin) [inline examples/wallet-export/src/components/ImportWallet.tsx:68]: nit: maybe add a CTA like `Enter Secret Recovery Phrase`?

File: .changeset/strong-hounds-work.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@turnkey/http": minor
+---
+
+Update to v2024.2.1 API: add InitImportWallet and ImportWallet activities
```
Comment (r-n-o) [inline .changeset/strong-hounds-work.md:5]: We should add the new delete endpoints to the changelog: delete users, delete private key tags, delete user tags: all possible now!

Comment (r-n-o) [reply .changeset/strong-hounds-work.md:5]: Oh! And we have a new `list_suborgs` endpoints to list sub organizations by email, public key, or credential ID!

**Review by r-n-o — COMMENTED** [review]
If we're using this PR as a post-release update, feel free to run `pnpm run changeset version`, and `pnpm run version` to update the versions and add to the package's changelogs.

Otherwise you'll need another PR to publish packages once this one merges.

File: .changeset/strong-hounds-work.md (hunk 3 above)
Comment (oliviathet) [reply .changeset/strong-hounds-work.md:5]: oh awesome, i didn't realize when i made the changeset! i'll add those now

File: packages/iframe-stamper/src/index.ts [hunk 4]
```diff
@@ -230,6 +239,65 @@ export class IframeStamper {
...
+          if (event.data?.type === IframeEventType.Error) {
+            reject(event.data["value"]);
+          }
+        },
+        false
+      );
+    });
+  }
+
+  async extractWalletEncryptedBundle(): Promise<string> {
```
Comment (r-n-o) [inline packages/iframe-stamper/src/index.ts:272]: I think this is tightly coupled to this discussion: https://github.com/tkhq/frames/pull/25#discussion_r1506786320. But I'm happy to get this merge as-is and change this as a followup (if we decide to change it, that is)

**Review by r-n-o — APPROVED** [review]

**Pull request merged**